flink-python/pyflink/table/table_config.py

```diff
@@ -44,18 +47,55 @@ def parallelism(self):
     def parallelism(self, parallelism):
         self._parallelism = parallelism
 
+    @property
+    def timezone(self):
+        return self._j_table_config.getTimeZone().getID()
+
+    @timezone.setter
+    def timezone(self, timezone_id):
+        if timezone_id is not None and isinstance(timezone_id, str):
```

Comment:
Will it be better to also support the Python timezone object as the argument?

Reply:
It's there some simple way to translate the python timezone to java timezone?

Reply:
The reason of using string here is that timezone class in python, `tzinfo` is a abstract class. It means that users could implements a custom timezone class by themselves. We can't  translate the custom python timezone to java timezone without information loss. So the easier way is using timezone ID in [tz database](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones). It is supported both in python and java.

Reply:
OK. That makes sense to me.